### DISCUSSION TITLE
Sling Player remote loses connection after 5 minutes or so

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#517] [General]
Hi Gerry,

I’ve noticed several times now, after i start sling player, start a stream to my sling server, i click the remote button and everything works fine.  after about 5 minutes the remote window turns a white background with a message about url cannot be found.   The video stream keeps playing.  if i close the remote window and click the remote button on the player, the remote comes back.  

Also, a side note, the remote window is small, height wise, for my remote, i need to resize the window each time.  not a big deal.

as always, appreciate your work.

**Comment by GerryDazoo** [comment]
Yes this appears to be a bug. I'll look into it.
What remote file are you using? You could make the remote page smaller so you don't need to resize the window.
I haven't been able to figure out how to resize the remote window to fit the page.

PS. It's SLINGER Player not sling player.

**Comment by GerryDazoo** [comment]
try this
[SlingerPlayer.zip](https://github.com/GerryDazoo/Slinger/files/10578786/SlingerPlayer.zip)

**Reply by pfuller88** [reply]
thanks, I will try now and report back.   I'm using my custom remote that ends in "PF".  It does have 3 sizes, the window size is not a big deal.

**Reply by pfuller88** [reply]
Running for the past 20 minutes or so and no disconnection for remote, working so far.

**Reply by tom76017** [reply]
Why is my Win11 detecting a Trojan?
![image](https://user-images.githubusercontent.com/26171888/216639543-c47e1e00-6deb-4677-b050-0f85afbc23e0.png)

**Reply by pfuller88** [reply]
My windows system did as well, when trying to download the above zip file.   Windows would not allow the doanload.

**Reply by GerryDazoo** [reply]
I assume you added an exception. I suspect it's because it's not a "signed" app. It's safe.

**Reply by pfuller88** [reply]
i had to turn windows defender off for a minute, then download.   i did a scan afterward with no issues.
